dashboard/src/app/diagnostics/test/diagnostics-workspace-start-check.factory.ts

```diff
@@ -140,16 +173,17 @@ export class DiagnosticsWorkspaceStartCheck {
           'diagnostics': {
             'machines': {
               'dev-machine': {
-                'agents': ['org.eclipse.che.ws-agent'],
+                'installers': ['org.eclipse.che.ws-agent'],
                 'servers': {},
                 'attributes': {'memoryLimitBytes': '1147483648'}
               }
             },
             'recipe': {
-              'content': 'FROM openjdk:8-jre-alpine\nCMD tail -f /dev/null\n',
+              'content': 'FROM openjdk:8-jre-alpine\nRUN apk add --update bash\nCMD tail -f /dev/null\n',
```

Comment:
it's not linked directly to this PR why do we need bash in the docker image? it seems to be a regression from che platform.

Reply:
Yes, this line should be reverted when #7124 is fixed